### PR TITLE
Big update to en_us.lua

### PR DESCRIPTION
**Pull request by Mirades**
- Added a lot of icons
- Changed the stat format slightly (made it consistent too)
- Updated upper/lower letter case
- More consistent wording (for example: drop/spawn and fire/shoot and shots/tears)
- Changed the order of stat increases and pickups to be consistent*
- Changed some descriptions

*oriented myself at this list:
↑ Life
↑ Health
↑ Empty heart container
↑ Damage (depending on the item: if the multiplier has impact on the damage up; first flat damage, then multiplier, if not; swap it) ↑ Tears
↑ Fire rate
↑ Tear delay (AB+)
↑ Range
↑ Tear height (AB+)
↑ Shot speed
↑ Tear size
↑ Speed
↑ Luck
↑ Size
↑ Bomb damage
↓ Negative stats in the same order
Heal
spawn or add random, Red, Soul, Black, Eternal, Bone Hearts
spawn or add Coins
spawn or add Bombs
spawn or add Keys
spawn Rune
spawn Card
spawn Pill
Devil, Angel chance main tear effect
main other effect
other effects
special conditions: starting again from top

PS: en_us.lua (rep) will get the same updates

### CODE REVIEW

**Comment by Mirades** [comment]
new order orientation:

↑ Life
↑ Health
↑ Empty heart container
↑ Speed
↑ Tears
↑ Fire rate (REP)
↑ Tear delay (AB+)
↑ Damage (depending on the item: if the multiplier has impact on the damage up; first flat damage, then multiplier, if not; swap it)
↑ Range
↑ Tear height (AB+)
↑ Shot speed
↑ Luck
↑ Tear size
↑ Size
↑ Bomb damage
↓ Negative stats in the same order
Heal
+ random, Red, Soul, Black, Eternal, Bone Hearts
+ Coins
+ Bombs
+ Keys
+ Rune
+ Card
+ Pill
+ Devil, Angel chance
main tear effect
main other effect
other effects
special conditions: starting again from top

**Comment by Mirades** [comment]
Github doesn't handle the 2 items i've removed well (Experimental Treatment and Toxic Shock).
It shows like 10-20 lines as completely rewritten...

Btw I've added the extra line for Car Battery effect that got implemented in the meantime.

**Comment by Mirades** [comment]
done with the big en_us.lua update.

feel free to discuss the changes with me, i'm active on discord (just ping me @Mirades).

File: descriptions/ab+/en_us.lua [hunk 1]
```diff
@@ -21,473 +21,473 @@ EID.descriptions[languageCode].fonts = {{name="default"}, {name="inverted"}, {na
...
-	{"233", "Tiny Planet", "Isaac's tears orbit around him#↑ {{Range}} Range way up#↑ +7 Tear Height#Spectral tears"},
+	{"222", "Anti-Gravity", "↑ {{Tears}} -2 Tear delay#Holding the fire buttons causes tears to hover in midair#Releasing the fire buttons shoots them in the direction they were fired"},
+	{"223", "Pyromaniac", "{{Bomb}} +5 Bombs#{{Heart}} Getting hit by explosions heals 1 heart instead of damaging Isaac#Immune to rock wave attacks and Mom and Satan's stomp attacks"},
+	{"224", "Cricket's Body", "↑ {{Tears}} -1 Tear delay#↑ {{Tearsize}} x1.2 Tear size#↓ {{Range}} -10 Range#Tears split up in 4 on hit#Split tears deal half damage"},
+	{"225", "Gimpy", "{{SoulHeart}} Taking damage has a chance to spawn a Soul Heart#{{HalfHeart}} Enemies have a chance to drop a half Red Heart on death"},
+	{"226", "Black Lotus", "↑ {{Heart}} +1 Health#{{SoulHeart}} +1 Soul Heart#{{BlackHeart}} +1 Black Heart"},
+	{"227", "Piggy Bank", "{{Coin}} +3 Coins#{{Coin}} Taking damage spawns 1-2 coins#{{Player14}} 0-1 Coin as Keeper"},
+	{"228", "Mom's Perfume", "↑ {{Tears}} -1 Tear delay#{{Fear}} 15% chance to shoot fear tears#{{Luck}} 100% chance at 85 luck"},
```
Comment (wofsauge) [inline descriptions/ab+/en_us.lua:251]: {{Luck}} 100% chance at 85 luck"
those look to me like useless information for the average player

File: descriptions/rep/en_us.lua [hunk 2]
```diff
@@ -1,424 +1,421 @@
...
-	[262] = {"262", "Missing Page 2", "{{BlackHeart}} +1 Black Heart#Taking damage down to 1 Heart damages all enemies in the room#{{Collectible35}} Black Hearts and Necronomicon effects deal double damage"}, -- Missing Page 2
-	[263] = {"263", "Clear Rune", "{{Rune}} Triggers the effect of the rune Isaac holds without using it#Drops 1 random rune on pickup"}, -- Clear Rune (REPENTANCE ITEM)
+	[205] = {"205", "Sharp Plug", "{{Battery}} Using an uncharged active item fully recharges it at the cost of half a heart per missing charge#{{Heart}} Removes Red Hearts first"}, -- Sharp Plug
+	[206] = {"206", "Guillotine", "↑ {{Tears}} +0.5 Fire rate#↑ {{Damage}} +1 Damage#Isaac's head becomes an orbital that shoots, doesn't take damage and deals 65 contact damage per second"}, -- Guillotine
+	[211] = {"211", "Spiderbaby", "Taking damage spawns 3-5 blue spiders"}, -- Spiderbaby
+	[214] = {"214", "Anemic", "↑ {{Range}} +1.5 Range#{{Timer}} When taking damage Isaac leaves a trail of blood creep for the room"}, -- Anemic
+	[218] = {"218", "Placenta", "↑ {{Heart}} +1 Health#{{Heart}} Heals 1 heart#{{HalfHeart}} 50 chance to heal half a heart every minute"}, -- Placenta
```
Comment (wofsauge) [inline descriptions/rep/en_us.lua:98]: "%" is missing

File: descriptions/rep/en_us.lua [hunk 3]
```diff
@@ -1,424 +1,421 @@
...
-	[370] = {"370", "Mr. Dolly", "↑ {{Tears}} +0.7 Tears up#↑ {{Range}} +2.5 Range up#{{Heart}} Spawns 3 random hearts"}, -- Mr. Dolly
-	[374] = {"374", "Holy Light", "10% chance to shoot Holy tears, which spawn a beam of light on hit#{{Damage}} The beams deals 3x Isaac's damage#{{Luck}} 50% chance at 9 Luck"}, -- Holy Light
-	[375] = {"375", "Host Hat", "Grants immunity to explosions#Grants immunity to falling projectiles#25% chance to reflect enemy shots"}, -- Host Hat
+	[369] = {"369", "Continuum", "↑ {{Range}} +3 Range#Spectral tears#Tears can travel through one side of the screen and come out the other side"}, -- Continuum
+	[370] = {"370", "Mr. Dolly", "↑ {{Tears}} +0.7 Tears#↑ {{Range}} +2.5 Range#{{{UnknownHeart}} Spawns 3 random hearts"}, -- Mr. Dolly
+	[374] = {"374", "Holy Light", "10% chance to shoot holy tears, which spawn a beam of light on hit#{{Damage}} The beams deals 3x Isaac's damage#{{Luck}} 50% chance at 9 luck"}, -- Holy Light
+	[375] = {"375", "Host Hat", "Immune to explosions and falling projectiles#25% chance to reflect enemy shots"}, -- Host Hat
```
Comment (wofsauge) [inline descriptions/rep/en_us.lua:170]: Same as in AB+: i like the old description better

File: descriptions/rep/en_us.lua [hunk 4]
```diff
@@ -1,424 +1,421 @@
...
-	[514] = {"514", "Broken Modem", "Causes some enemies and projectiles to briefly freeze at random intervals#Frozen projectiles are deleted#25% chance to double room clear drops"},
+	[506] = {"506", "Backstabber", "{{BleedingOut}} Hitting an enemy in the back deals double damage and causes bleeding, which makes enemies leave creep and take damage when they move"},
+	[507] = {"507", "Sharp Straw", "{{Damage}} Deals Isaac's damage + 10% of the enemy's max health to all enemies#{{HalfHeart}} Dealing damage with the Straw can spawn half hearts"},
+	[508] = {"508", "Mom's Razor", "{{BleedingOut}} Orbital that causes bleeding, which makes enemies take damage when they move#{{Damage}} Deals 1.5x Isaac's damage per second"},
+	[509] = {"509", "Bloodshot Eye", "Orbital that shoots a tear every 0.33 seconds to nearby enemies#Deals 3.5 damage per tear#Deals 20 contact damage per second"}, -- Bloodshot Eye
+	[514] = {"514", "Broken Modem", "Causes some enemies and projectiles to briefly pause at random intervals#Paused projectiles disappear#25% chance to double room clear rewards#(Doesn't double chest, card or pill rewards)"},
```
Comment (wofsauge) [inline descriptions/rep/en_us.lua:231]: Same issue as in AB+

File: descriptions/rep/en_us.lua [hunk 5]
```diff
@@ -1,424 +1,421 @@
...
 	[576] = {"576", "Dirty Mind", "All Dip (small poop) enemies are friendly#Destroying poop spawns 1-4 Dips#Dip type depends on the poop type#Rocks may be replaced with poop"}, -- Dirty Mind
-	[577] = {"577", "Damocles", "{{Warning}} SINGLE USE {{Warning}}#Hangs a sword above Isaac's head, which doubles all pedestal items#Does not double Shop, Chest, or Devil deal items#{{Warning}} After taking any damage, the sword has a chance to instantly kill Isaac every frame"}, -- Damocles
-	[578] = {"578", "Free Lemonade", "Creates a large pool of yellow creep"}, -- Free Lemonade
-	[579] = {"579", "Spirit Sword", "Instead of shooting tears, Isaac swings a sword#The sword deals 3x damage#{{Chargeable}} Charging does a spin attack + projectile shot#Fire projectiles with normal swings at full health"}, -- Spirit Sword
+	[577] = {"577", "Damocles", "!!! SINGLE USE !!!#Hangs a sword above Isaac's head, which doubles all pedestal items#Does not double shop, chest, or devil deal items#!!! After taking any damage, the sword has an extremely low chance to instantly kill Isaac every 0.133 seconds"}, -- Damocles
```
Comment (wofsauge) [inline descriptions/rep/en_us.lua:263]: 0.133 seconds is way to specific and not really "graspable" for every player. Frame is more understandable i think

File: descriptions/rep/en_us.lua [hunk 6]
```diff
@@ -474,222 +471,222 @@ EID.descriptions[languageCode].birthright ={
...
+	[475] = "Middle ring wisp#Spawns 8 high damage wisps that stay after death", -- Plan C
+	[476] = "Middle ring wisp#Spawns a second wisp if no pickup was duplicated", -- D1
 	[477] = "Spawns the wisps of all absorbed active items#Doesn't spawn a wisp itself", -- Void
-	[478] = "Pauses all enemies and shots for 3 seconds when destroyed", -- Pause
-	[479] = "Chance to spawn a trinket when destroyed", -- Smelter
-	[480] = "Wisp's health increases the more pickups were destroyed", -- Compost
-	[481] = "5% chance for tears to transform enemies into random wisps", -- Dataminer
+	[478] = "Middle ring wisp#Pauses all enemies and shots for 3 seconds when destroyed", -- Pause
+	[479] = "Middle ring wisp#Chance to spawn a {{Trinket}} trinket when destroyed", -- Smelter
+	[480] = "Middle ring wisp#Only spawns a wisp if at least one pickup got converted#Increased HP for each pickup converted", -- Compost
```
Comment (wofsauge) [inline descriptions/rep/en_us.lua:623]: change order of informations, to first show the HP increase, and then the limitations

**Review by wofsauge — CHANGES_REQUESTED** [review]
Please fix the commented issues,

ALSO:
revert the change of the "{{Warning}}" markup to "!!!". Using "{{Warning}}" is easier for new translators to identify as Markup relevant content, which makes it less likely for them to delete those.

**Pull request merged**